cli/src/main/java/com/devonfw/tools/ide/tool/LocalToolCommandlet.java

```diff
@@ -60,13 +62,14 @@ public Path getToolBinPath() {
   protected boolean doInstall(boolean silent) {
```

Comment:
This changes shall fix the case that the first installation is always the current "resolved" installation. This makes it impossible to call the postInstall method if try to force an installation or if it's the first installation-

Reply:
Thanks for finding and fixing this problem. 👍 
I will set this conversation as resolved.